packages/core/helper-plugin/lib/src/components/RelationsInput/RelationsInput.stories.mdx

```diff
@@ -0,0 +1,54 @@
+<!--- RelationsInput.stories.mdx --->
+
+import { Meta, ArgsTable, Canvas, Story } from '@storybook/addon-docs';
+import { BaseLink } from '@strapi/design-system/BaseLink';
+import { Typography } from '@strapi/design-system/Typography';
+import { Badge } from '@strapi/design-system/Badge';
+import { Icon } from '@strapi/design-system/Icon';
+import Cross from '@strapi/icons/Cross';
+import { ShadowList, RelationItem } from './index';
+import { relations } from './relationsDataMock';
+
+<Meta title="components/RelationsInput" />
```

Comment:
What do you think about calling the component `RelationInput` instead for consistency? The other ones always refer to a single instance (e.g. `NumberInput`).

Reply:
instead of `RelationsInput`? sure!